### PR TITLE
if dash is enabled, auto-configure policy record

### PR DESCRIPTION
**Pull request by asoorm**
Addresses #1505

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -833,6 +833,13 @@ func initialiseSystem() error {
 		if globalConf.PIDFileLocation == "" {
 			globalConf.PIDFileLocation = "/var/run/tyk/tyk-gateway.pid"
 		}
+		if globalConf.UseDBAppConfigs && globalConf.Policies.PolicySource != "service" {
+			globalConf.Policies.PolicySource = "service"
+			globalConf.Policies.PolicySource = globalConf.DBAppConfOptions.ConnectionString
```
Comment (dencoded) [inline main.go]: looks like it is supposed to be `globalConf.Policies.PolicyConnectionString `

File: main.go [hunk 2]
```diff
@@ -833,6 +833,13 @@ func initialiseSystem() error {
 		if globalConf.PIDFileLocation == "" {
 			globalConf.PIDFileLocation = "/var/run/tyk/tyk-gateway.pid"
 		}
+		if globalConf.UseDBAppConfigs && globalConf.Policies.PolicySource != "service" {
+			globalConf.Policies.PolicySource = "service"
```
Comment (dencoded) [inline main.go]: might worth to introduce constants for for string values like `"service"` and `"tyk_policies"`, maybe in config package (not sure about that)

File: main.go (hunk 1 above)
Comment (asoorm) [reply main.go]: good spot

**Comment by buger** [comment]
Merged manually

**Pull request closed without merging**